roslibrust/src/rosbridge/mod.rs

```diff
@@ -94,64 +94,41 @@ impl ClientOptions {
     }
 }
 
-/// A client connection to the rosbridge_server that allows for publishing and subscribing to topics
 #[derive(Clone)]
 pub struct Client {
```

Comment:
Seperate todo and probably worth a new issue, but client should be in its own file

Reply:
Took a quick swing at this and it took more than a minute so I'll pick it up in another branch.